Resources/public/js/views/fields/ez-selection-editview.js

```diff
@@ -330,6 +330,18 @@ YUI.add('ez-selection-editview', function (Y) {
         },
 
         /**
+         * Check if there is a value that has been added or removed with the event facade.
+         * If it is, then call validate().
+         *
+         * @method checkValueChange
+         */
+        checkValueChange: function (e) {
```

Comment:
this should be a protected method (tag in the jsdoc + `_`) and maybe called `_validateAddedRemovedValues`